roles/backup/tasks/postgres.yml

```diff
@@ -82,6 +82,16 @@
     resolvable_db_host: '{{ (awx_postgres_type == "managed") | ternary(awx_postgres_host + "." + ansible_operator_meta.namespace + ".svc.cluster.local", awx_postgres_host) }}'  # yamllint disable-line rule:line-length
   no_log: "{{ no_log }}"
 
+- name: Precreate database partitions
+  k8s_exec:
+    namespace: '{{ ansible_operator_meta.namespace }}'
+    pod: "{{ tower_pod_name }}"
```

Comment:
just do double check, can you show me where is this var defined?

Reply:
I think I carried over a name from automation-platform-collection when I was working on both PRs at the same time. Not sure where I pulled this from but it's been updated to "{{ awx_task_pod_name }}" which I believe is defined in the resources_configuration.yml